### DISCUSSION TITLE
Neuer Parameter für eine Task: `pathSuggestion`

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#174] [Ideas]
Das wäre im Prinzip das gleiche wie `stations`, nur dass die Punkte nicht zwingend Halte beschreiben müssen, sondern nur für die vorausgewählten Haltepunkte relevant sind.  
Beispiel:
```json
{
	"name": "ICE von %s nach %s",
	"descriptions": [],
	"neededCapacity": [
		{ "name": "passengers", "value": 0 }
	],
	"group": 1,
	"plops": 436000,
	"stations":       [ "AH", "AHAR",               "HH", "HG", "FKW", "FFU",               "NWH",               "NN", "MIH", "MH" ],
	"pathSuggestion": [ "AH", "AHAR", "ALBG", "HU", "HH", "HG", "FKW", "FFU", "NBN", "NRB", "NWH", "NRTD", "NF", "NN", "MIH", "MH" ]
}
```

**Comment by marhei** [comment]
Wird umgesetzt, du kannst das gerne schonmal in Schema und eigene TaskModelle implementieren.

**Reply by marhei** [reply]
Ist umgesetzt, viel Spaß beim einarbeiten!

**Reply by C1710** [reply]
Ich denke, ich werde morgen mein Script erweitern, dass es das auch für bestehende Strecken hinzufügen kann...
